src/main/drivers/pwm_output.c

```diff
@@ -460,8 +456,8 @@ void servoDevInit(const servoDevConfig_t *servoConfig)
             /* flag failure and disable ability to arm */
             break;
         }
-
-        pwmOutConfig(&servos[servoIndex], timer, PWM_TIMER_MHZ, 1000000 / servoConfig->servoPwmRate, servoConfig->servoCenterPulse, 0);
+        const uint16_t period = PWM_TIMER_HZ / servoConfig->servoPwmRate;
+        pwmOutConfig(&servos[servoIndex], timer, PWM_TIMER_HZ, period, (servoConfig->servoCenterPulse - 1000) / 1000 * period, 0);
```

Comment:
then it was correct as it was, as the timer is running at 1mhz, thus the ticks are 1 per us.

PWM_TIMER_HZ is 1000000. Thats not going to change.